docker-compose.yml

```diff
@@ -112,9 +114,11 @@ services:
       - TOKEN_LIST
       - UPHOLD_ACCESS_TOKEN
       - GEMINI_SERVER
-      - GEMINI_CLIENT_KEY
+      - GEMINI_API_KEY
       - GEMINI_CLIENT_SECRET
       - GEMINI_CLIENT_ID
+      - GEMINI_BROWSER_CLIENT_ID
+      - GEMINI_API_SECRET
```

Comment:
Nit picking, I believe the values should be grouped together

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/881/commits/16ce255a704425f8dcec556994fbbee8ca667fb2